Fauna-RPG/Buffs/Buff_Poison.gd

```diff
@@ -0,0 +1,13 @@
+extends BuffBase
+class_name Buff_Poison
+## Poison deals damage to the owning Entity at the start of their turn
+
+
+func on_turn_start() -> void:
```

Comment:
Yeah I find it a bit annoying that there's no keyword. Adding a standard comment when we override would be helpful

Reply:
Should we make the change now ? I think there are quite a lot of those and a comment doesn't really change code. I feel like it would be better to do it in a future PR probably

Reply:
Yes I'm fine with waiting until later